src/intl/resolver.js

```diff
@@ -257,6 +257,9 @@ function* toString(entity) {
 }
 
 export function format(ctx, args, entity) {
+  if (entity === undefined) {
+    return ['', []];
+  }
```

Comment:
I had a problem where a missing entity would break something. Need to test what was breaking to understand if that's the right fix.